src/history/hs.c

```diff
@@ -1216,8 +1272,10 @@ __wt_hs_find_upd(WT_SESSION_IMPL *session, WT_ITEM *key, const char *value_forma
          * we can skip it.
          */
         if (__wt_txn_visible_all(
-              session, hs_cbt->upd_value->tw.stop_txn, hs_cbt->upd_value->tw.durable_stop_ts))
+              session, hs_cbt->upd_value->tw.stop_txn, hs_cbt->upd_value->tw.durable_stop_ts)) {
+            ret = __wt_hs_cursor_prev(session, hs_cursor);
```

Comment:
Is this different to the old version that just had the call in the increment for the for loop?

Reply:
No. I think this was done because it was originally a macro.
I should have restructured the loop as part of the change to make them function calls.

Reply:
aedc9c4